thirdeye-integration-tests/src/test/java/ai/startree/thirdeye/SchedulingTest.java

```diff
@@ -57,7 +57,7 @@
  * - wait for detection task to run. Check lastTimestamp
  *
  * Note: if run within IntelliJ, run with the following JVM option:
- * -javaagent:[USER_PATH]/.m2/repository/org/aspectj/aspectjweaver/1.9.6/aspectjweaver-1.9.6.jar
+ * -javaagent:[USER_PATH]/.m2/repository/org/aspectj/aspectjweaver/1.9.21/aspectjweaver-1.9.21.jar --add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/java.time=ALL-UNNAMED --add-opens java.base/java.util=ALL-UNNAMED
```

Comment:
Just to understand, what is the issue here?

Reply:
AspectJ does bytecode manipulation and is only compatible with java 17 starting from `1.9.8`, so I had to upgrade.
I took `1.9.21`, it's the most recent one and should only include minor fixes on top of `1.9.8`.